Ladybird/AppKit/main.mm

```diff
@@ -4,25 +4,42 @@
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
+#include <BrowserSettings/Defaults.h>
 #include <Ladybird/Utilities.h>
 #include <LibCore/ArgsParser.h>
 #include <LibCore/EventLoop.h>
 #include <LibGfx/Font/FontDatabase.h>
 #include <LibMain/Main.h>
 #include <LibWebView/CookieJar.h>
 #include <LibWebView/Database.h>
+#include <LibWebView/URL.h>
 
 #import <Application/Application.h>
 #import <Application/ApplicationDelegate.h>
 #import <Application/EventLoopImplementation.h>
 #import <UI/Tab.h>
 #import <UI/TabController.h>
-#import <Utilities/URL.h>
 
 #if !__has_feature(objc_arc)
 #    error "This project requires ARC"
 #endif
 
+static URL rebase_url_on_serenity_resource_root(StringView url_string)
```

Comment:
I wonder if this will be necessary once I get https://github.com/SerenityOS/serenity/pull/21302 working for app bundle usage?

Reply:
I was wondering that too - haven't not looked too closely at it, it does seem like this can go away after 21302